### ISSUE TITLE
Favicon

### ISSUE THREAD

**Issue by crunchylian** [dAnjou/deutsch#3]
Add a favicon to spot the website.

**Comment by dAnjou** [comment]
Any idea what it's gonna be?

**Comment by crunchylian** [comment]
A German flag?

**Comment by dAnjou** [comment]
I don't want to promote countries on the website.

It's a good idea in general but not having a favicon is not a big deal for this website. A favicon's purpose is to help the user to quickly find a website amongst their tabs or bookmarks. This isn't needed for this website because the website's primary purpose is to contain the rules of the channel. If you're not in the channel then the website should be totally irrelevant.

**Comment by crunchylian** [comment]
Well, it could be sth else. But I don't see the point of listing resources if you don't use them. I had the tab pinned. And since it's not the only one, I hate to search sth when all I can see is identical favicons. It's not #firstproblemintheworld, but still I always like a little favicon.h

**Comment by dAnjou** [comment]
Okay, let me try again as I've clearly not get the point across yet.

You should not rely on the website. It was never intended for people to pin or bookmark it. The website is not supposed to be a central entry point for learning German. It's purpose is not to attract people, it's not a marketing website for the channel. It's not supposed to be a tool to build a bigger community. The original purpose was and still is to be a supportive document for the channel, like, an extension of the channel topic because putting all that information (including the rules now) in the channel topic doesn't make sense and might not even be possible.

So, feel free to extract useful information from the website into your own bookmarks but don't rely on the website. It might be taken down anytime or moved somewhere else or its content might change from one day to the next without notice.

**Comment by crunchylian** [comment]
Thank you, I understand your point of view. Putting the chan first makes sense, indeed. I am fairly new to the chan, I'll back off on this one. ;) Also, far from me the idea of making a marketing tool. I came to this chan because of free software (and I am a moderator on Ubuntu-fr, we remind people daily that -fr stand for French, not for France, so I'm quite receptive to your argument).
Still I disagree on one points. :D
« The website is not supposed to be a central entry point for learning German. » ← for someone coming from Freenode, it is a central entry point, especially when all language chans begin with ##. I respect the fact that you (or some friends) have founded it. But the chan is de facto an entry point on Freenode.

**Comment by dAnjou** [comment]
> Also, far from me the idea of making a marketing tool.

Didn't mean to imply that, just wanted to put things into perspective.

> « The website is not supposed to be a central entry point for learning German. » ← for someone coming from Freenode, it is a central entry point, especially when all language chans begin with ##. I respect the fact that you (or some friends) have founded it. But the chan is de facto an entry point on Freenode.

Yes, the channel is but not the website ;)
